scripts/logfetch/entrypoint.py

```diff
@@ -81,4 +85,47 @@ def entrypoint():
 
   args.dest = os.path.expanduser(args.dest)
 
-  main(args)
+  fetch_logs(args)
+
+def tail():
+  conf_parser = argparse.ArgumentParser(description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter, add_help=False)
+  conf_parser.add_argument("-f", "--conf_folder", help="specify a folder for config files to live")
+  conf_parser.add_argument("-c", "--conf_file", help="Specify config file within the conf folder", metavar="FILE")
+  args, remaining_argv = conf_parser.parse_known_args()
+  conf_dir = args.conf_folder if args.conf_folder else DEFAULT_CONF_DIR
+  conf_file = os.path.expanduser(conf_dir + '/' + args.conf_file) if args.conf_file else os.path.expanduser(conf_dir + '/' + DEFAULT_CONF_FILE)
+  config = ConfigParser.SafeConfigParser()
+
+  defaults = {"start_days" : DEFAULT_DAYS}
+
+  try:
+    config.readfp(FakeSectionHead(open(os.path.expanduser(conf_file))))
+    defaults.update(dict(config.items("Defaults")))
+  except Exception, err:
+    sys.stderr.write(CONF_READ_ERR_FORMAT.format(conf_file, err) + '\n')
+
+  parser = argparse.ArgumentParser(parents=[conf_parser], description="Tail log files from Singularity. One can specify either a TaskId, RequestId and DeployId, or RequestId",
+          prog="log_fetcher")
+
+  parser.set_defaults(**defaults)
+  parser.add_argument("-t", "--taskId", help="TaskId of task to fetch logs for", metavar="taskId")
+  parser.add_argument("-r", "--requestId", help="RequestId of request to fetch logs for", metavar="requestId")
+  parser.add_argument("-d", "--deployId", help="DeployId of task to fetch logs for", metavar="deployId")
+  parser.add_argument("-u", "--singularity-uri-base", help="The base for singularity (eg. http://localhost:8080/singularity/v1)", metavar="URI")
+  parser.add_argument("-s", "--start-days", help="Search for logs no older than this many days", type=int, metavar="start_days")
+  parser.add_argument("-e", "--end-days", help="Search for logs no new than this many days (defaults to None/today)", type=int, metavar="end_days")
+  parser.add_argument("-g", "--grep", help="String to grep for", metavar='grep')
+  parser.add_argument("-l", "--logfile", help="Logfile path/name to tail (ie 'logs/access.log')", metavar="logfile")
```

Comment:
There isn't a default for it currently, right now I have it providing an error message if the log to tail isn't set